lib/htsget/docker-compose.yml

```
@@ -21,7 +25,7 @@ services:
       HTSGET_URL: ${TYK_LOGIN_TARGET_URL}/${TYK_HTSGET_API_LISTEN_PATH}
       OPA_URL: ${OPA_PRIVATE_URL}
       VAULT_URL: ${VAULT_PRIVATE_URL}
-      DEBUG_MODE: ${CANDIG_DEBUG_MODE}
+      DEBUG_MODE: 0 #${CANDIG_DEBUG_MODE}
```

```suggestion
      DEBUG_MODE: ${CANDIG_DEBUG_MODE}
```
Was this for testing?

---

Annoyingly, connexion has a bunch of debug-level logging built into it that I can't turn off, so I did this. I'm not sure what else to do about it.